doc/api/util.md

```diff
@@ -198,7 +198,7 @@ Each placeholder token is replaced with the converted value from the
 corresponding argument. Supported placeholders are:
 
 * `%s` - `String`.
-* `%d` - `Number` (integer or floating point value).
+* `%d` - `Number` (integer or floating point value) or `BigInt`.
```

Comment:
If anything, shouldn't `%i` be more suitable for `BigInt`, considering it's an integer?